README.md

```diff
@@ -12,10 +12,9 @@ The extension uses [macro_railroad](https://github.com/lukaslueg/macro_railroad)
 
 To modify and build the extension locally:
 
-* Install the `cargo-web` binary from `crates.io`.
-* The package in `wasm_src` can be executed via `cargo web start`; it serves a testing-page for a quick edit-debug-cycle.
-* The javascript-shim built by `cargo-web` currently needs a little patching which the `Makefile` takes care of.
-  * Building requires `wasm-opt` (from `binaryen`). If `wasm-opt` is not available, set `$TRAVIS` to some value.
+* Install the `wasm-pack` binary from `crates.io`.
+* The package in `wasm_src` can be executed via `wasm-pack build -t no-modules` to build destination wasm package.
+* The javascript-shim built by `wasm-pack` currently needs a little patching which the `Makefile` takes care of.
```

Comment:
Can be removed, this was referring to the `sed` business in the `Makefile`, which has been removed. I'm unsure if the patching is actually still required to make the extension work under Firefox (it shouldn't). In any case, the extension is currently rejected by Firefox for other unknown reasons (probably manifest v2 deprecation)